packages/Paragraph/Paragraph.jsx

```diff
@@ -1,38 +1,42 @@
 import React from 'react'
 import PropTypes from 'prop-types'
+import styled from 'styled-components'
 
-import typographyStyles from '@tds/shared-typography/Typography.modules.scss'
+import * as typography from '@tds/shared-typography'
+import { spacing } from '@tds/shared-styles'
 
 import safeRest from '../../shared/utils/safeRest'
-import joinClassNames from '../../shared/utils/joinClassNames'
 
-import spacingStyles from '../../shared/styles/Spacing.modules.scss'
-import styles from './Paragraph.modules.scss'
+const paragraphColor = ({ invert }) => (invert ? typography.invertedColor : typography.color)
+const paragraphInheritColor = ({ inheritColor }) =>
+  inheritColor ? { color: 'inherit' } : undefined
+
+const paragraphSize = ({ size }) => typography[size]
+const paragraphBold = ({ bold, size }) => (bold ? typography.boldFont : typography[`${size}Font`])
+const paragraphAlign = ({ align }) => ({
+  textAlign: align,
+})
+
+export const StyledParagraph = styled.p(
```

Comment:
I've just noticed that `sup` styles are missing here.
Previously these styles were coming from `typographyStyles.default` in the new shared typography this style object is called `sup`.